test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,27 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+    }
+
+    @Test
+    public void elementsAreNull() throws Exception {
+
+        // empty list
+        assertNoNull();
+
+        // contain one null object
+        assertContainsNull((Object) null);
+
```

Comment:
These JUnit tests should not be part of this PR.  Don't mix multiple enhancements in the same PR. Merge previous week's code to the master before branching for a new week's LOs.